src/main/java/org/sonatype/nexus/repository/composer/internal/ComposerGroupProviderJsonHandler.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2018-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.repository.composer.internal;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import org.sonatype.nexus.repository.Repository;
+import org.sonatype.nexus.repository.group.GroupFacet;
+import org.sonatype.nexus.repository.group.GroupHandler;
+import org.sonatype.nexus.repository.http.HttpResponses;
+import org.sonatype.nexus.repository.http.HttpStatus;
+import org.sonatype.nexus.repository.view.Context;
+import org.sonatype.nexus.repository.view.Payload;
+import org.sonatype.nexus.repository.view.Response;
+
+import org.joda.time.DateTime;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Handler for merging provider JSON files together.
+ */
+@Named
+@Singleton
+public class ComposerGroupProviderJsonHandler
+    extends GroupHandler
+{
+  private final ComposerJsonProcessor composerJsonProcessor;
+
+  @Inject
+  public ComposerGroupProviderJsonHandler(final ComposerJsonProcessor composerJsonProcessor) {
+    this.composerJsonProcessor = checkNotNull(composerJsonProcessor);
+  }
+
+  @Override
+  protected Response doGet(@Nonnull final Context context,
+                           @Nonnull final GroupHandler.DispatchedRepositories dispatched)
+      throws Exception
+  {
+    Repository repository = context.getRepository();
+    GroupFacet groupFacet = repository.facet(GroupFacet.class);
+    Map<Repository, Response> responses = getAll(context, groupFacet.members(), dispatched);
+    List<Response> successfulResponses = responses.values().stream()
+        .filter(response -> response.getStatus().getCode() == HttpStatus.OK && response.getPayload() != null)
```

Comment:
Similar question to the packages handler about conditional gets

Reply:
> Similar question to the packages handler about conditional gets

Done in https://github.com/sonatype-nexus-community/nexus-repository-composer/pull/14/commits/788b51a83c47eb12790bfb3b3b805fe86d1c4de6.